examples/4337-passkeys/README.md

```diff
@@ -8,7 +8,7 @@ This minimalistic example application demonstrates a Safe{Core} Smart Account de
 
 ```bash
 git clone https://github.com/safe-global/safe-modules.git
-cd safe-modules
+cd safe-modules/examples/4337-passkeys
```

Comment:
One must run the `pnpm install` (the next step in the readme) in the project's root directory. Running it in the package directory may work as well, but I'm not sure if pnpm handles nested directories correctly

Reply:
I think it should work: https://github.com/orgs/pnpm/discussions/4735#discussioncomment-2770033 but please test it :pray:

Reply:
If we change the `cd` directory here, I think we should update other commands to also not assume that they are run from the repository root. For example this one:

https://github.com/safe-global/safe-modules/blob/8b85c79b52454606a11b5857018af797ee8f1c3d/examples/4337-passkeys/README.md?plain=1#L35-L37